isis/src/control/objs/BundleResults/BundleResults.cpp

```diff
@@ -2313,11 +2328,11 @@ namespace Isis {
                           toString(maximumLikelihoodMedianR2Residuals()));
 
     stream.writeStartElement("cumulativeProbabilityCalculator");
-    // cumulativeProbabilityDistribution().save(stream, project);
+    cumulativeProbabilityDistribution().save(stream, project);
     stream.writeEndElement(); // end cumulativeProbabilityCalculator
 
     stream.writeStartElement("residualsCumulativeProbabilityCalculator");
-    // residualsCumulativeProbabilityDistribution().save(stream, project);
```

Comment:
Hey @kledmundson, I was meaning to email you about this but figured I would ask you here. Was there a reason that these distributions were commented out of the bundle result serialization? If there is I can revert this change but it seemed weird that it was commented out

Reply:
Sadly, I can't recall why/when/how this was done. I agree that it doesn't make sense to be removed specifically from IPCE. That being said, I'm not sure we ever finished properly serializing an IPCE project. Just for kicks I grabbed your code, built it, and ran a bundle both from the command line and in IPCE. Got identical results as expected. Then tried to save the IPCE project and it crashed on me. Don't know why. Based on where we were when we had to stop, I'm not completely sure we had everything serialized properly. Have you tested this in IPCE? And been able to save and restore a project with a bundle in it?

Reply:
I have not but there is a test for bundle results serialization that was failing on Mac and this addresses it. With the commented outlines, the `cumulativeProbabilityCalculator` and `residualsCumulativeProbabilityCalculator` elements in the XML where empty causing issues when trying to read the serialized bundle results back in.

It doesn't surprise me that IPCE crashed while saving a bundle/project as we made a number of changes to the XML writing updating to qt 5.15. I just don't know if the crash existed prior to that XML change since IPCE isn't something we poke at regularly.

Reply:
Then, I would say go ahead and merge it if only to get the tests passing. We If IPCE ever lives again we'll want those stats to persist in the project and the serialization will have to be re-sorted out anyway.